week-2/D-predicates/exercise.js

```diff
@@ -7,12 +7,12 @@
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
 function isNegative(number) {
-
+  return (number = false);
```

Comment:
How do you check if a number is negative?, that is what you should return, this is the same for the isBetweenZeroAnd10 function